### ISSUE TITLE
regExp does not match hiAscii characters

### ISSUE THREAD

**Issue by duffy-corelight** [zeek/spicy#328]
```
module Issue_328_no_hiAscii_regExp;

public type beginDissassembly = unit {
  : oneInstr[];
};
  
type oneInstr = unit {
    var lengthOfPrebytes: uint16;
   prebytes: /[\x7F\x80]*/;
   nonprebyte: bytes &size=1;   
    on nonprebyte {
      self.lengthOfPrebytes = |self.prebytes|;
      if (self.lengthOfPrebytes) {
        print ("prebyte %s" % (self.prebytes,));
      }
      print ("%s" % (self.nonprebyte,));
    }
};
```
when fed a mix of lo and hi Ascii characters, matches the \x7F but misses the \x80 outputs:
```
spicy/build/bin/spicy-driver /usr/local/etc/tmp/Issue_328_no_hiAscii_regExp.spicy  < x7fx80.bin 
\xc4
prebyte \x7f
\x80
\x80
\x80
\x81
\x80
\x82
```

**Comment by duffy-corelight** [comment]
my current executable, where I observe this, is: Apr 27 14:39 spicy/build/bin/spicy-driver
spicy-driver v0.4.0-154.dirty (master 51f4cd5)

**Comment by bbannier** [comment]
I filed #348 to check our support for regexp containing byte literals.

**Comment by rsmmr** [comment]
Closed along with #348.
